packages/blockchain-wallet-v4-frontend/src/components/Alerts/template.js

```diff
@@ -43,6 +43,13 @@ const Alerts = props => {
             persist={persist}
             onClose={() => handleClose(id)}
           >
+            {nature === 'success' ? (
```

Comment:
By adding this check here, now every single alert that is of nature `success` gets this checkmark icon.  Is this really what design wanted?

For example, now on language change gets the icon.
<img width="474" alt="Screen Shot 2019-09-17 at 9 45 21 AM" src="https://user-images.githubusercontent.com/6364918/65026322-3c9b9e80-d930-11e9-98fd-37c395bac039.png">

Also the green edge and blue icon seems strange... 🤔

Reply:
Yes, agreed wrt blue on green. 

Also, I assume the checkmark is actually only meant to appear on signup.

I could add it directly to the alert component (`src/components/Alerts/messages.js`) FormattedMessage for that case.